utils/emmylua/wesnoth/units.lua

```diff
@@ -289,4 +289,15 @@ function wesnoth.units.create_weapon(cfg) end
 ---@overload fun(unit:unit, x:integer, y:integer, ignore_passability:boolean, clear_shroud:boolean, animate:boolean)
 function wesnoth.units.teleport(unit, target, ignore_passability, clear_shroud, animate) end
 
+---Fakes the move of a given unit, can be queued.
+---Todo: move it to C++ side implementation.
+---@param u_a unit_animator
+---@param u unit
+---@param to_x integer
+---@param to_y integer
+---@param queued boolean
```

Comment:
`queued` is not part of the API, so it shouldn't be listed here.

To be clear, the function you're documenting here is the function you defined in C++, not the Lua function that said C++ function calls. In other words, this is the documentation for `impl_add_movement` from the perspective of Lua. It is _not_ the documentation for `wesnoth.interface.move_unit_fake_queue`. The `impl_add_movement` function only takes at most 4 arguments (or 5 if the location is passed as separate x and y coordinates). Since it's called with `:`, there is also the invisible "self" argument (here called `u_a`), so that comes to 4 or 5 arguments, none of which are `queued`.